web/src/FatalErrorModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from "react"
+import Modal from "react-modal"
+import { ShowFatalErrorModal } from "./types"
+import "./FatalErrorModal.scss"
+
+type props = {
+  error: string | null
+  showFatalErrorModal: ShowFatalErrorModal
+  handleClose: () => void
+}
+
+export default class FatalErrorModal extends PureComponent<props> {
+  render() {
+    let showModal =
+      this.props.error !== null &&
```

Comment:
can we just use `this.props.error &&`?

i feel like we've had a lot of bugs due to bad use of strict equality. for example, in this case, `""` and `undefined` should not show the modal